README

```diff
@@ -1,3 +1,27 @@
+
+## Filestructure
+Path                                    | Comment
+--------------------------------------- | -------------
+`/desktop/src/com/mygdx/game/`          | File to run the game
+`/core/src/`                            | The main code folder 
+`/core/test/java/`                      | JUnit tests folder
+`/core/assets`                          | Location for all assets in the game
+
+
+
+## Creators
```

Comment:
I think this is unnecessary. Instead I think a basic description of the project should be added.